flopy/mf6/utils/codegen/dfn2toml.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+from pathlib import Path
+
+from flopy.utils import import_optional_dependency
+
+_MF6_PATH = Path(__file__).parents[2]
+_DFN_PATH = _MF6_PATH / "data" / "dfn"
+_TOML_PATH = _MF6_PATH / "data" / "toml"
+
+__tomlkit = import_optional_dependency("tomlkit")
```

Comment:
Can you explicitly state this dependency in the pyproject? This seems to be a transient dependency.

Reply:
Another option would be to use tomllib, although only added since python 3.11

Reply:
done, thanks